root/Makefile

```diff
@@ -238,6 +238,11 @@ benchmarker-run:
 deploy:
 	@$(MAGE_CMD) deploy $(APP) $(CHANNEL)
 
+## checkopslevel: checks if the app passes require opsevel checks
+.PHONY: checkopslevel
```

Comment:
The catch-all should do this, right? See L241

Reply:
I don't know why the deploy one was explicitly added here either tbh...

Reply:
But will it pass through the `$(APP)`? I thought that's why we created specific steps.

Reply:
But will it pass through the `$(APP)`? I thought that's why we created specific steps.

Reply:
Agh. Good point. I hate it :/

Reply:
Really the target should determine `APP` itself. Now we depend on `make` functionality here.